<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>669</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Zappy</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="mainTab">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>651</width>
      <height>401</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="serverWidget">
     <attribute name="title">
      <string>Server</string>
     </attribute>
     <layout class="QGridLayout" name="firstLayout">
      <item row="5" column="0" rowspan="2">
       <layout class="QGridLayout" name="serverLayout">
        <item row="1" column="5">
         <widget class="QLabel" name="mapWidthLabel">
          <property name="text">
           <string>Map Width: </string>
          </property>
         </widget>
        </item>
        <item row="1" column="6">
         <widget class="QSpinBox" name="mapWidthSpinbox">
          <property name="minimum">
           <number>10</number>
          </property>
          <property name="maximum">
           <number>100000</number>
          </property>
          <property name="value">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item row="0" column="5">
         <widget class="QLabel" name="serverPortLabel">
          <property name="text">
           <string>Server Port: </string>
          </property>
         </widget>
        </item>
        <item row="4" column="5">
         <widget class="QLabel" name="delayLabel">
          <property name="text">
           <string>Action delay:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="6">
         <widget class="QSpinBox" name="serverPortSpinbox">
          <property name="maximum">
           <number>100000</number>
          </property>
          <property name="value">
           <number>8080</number>
          </property>
         </widget>
        </item>
        <item row="2" column="5">
         <widget class="QLabel" name="mapHeightLabel">
          <property name="text">
           <string>Map Height: </string>
          </property>
         </widget>
        </item>
        <item row="2" column="6">
         <widget class="QSpinBox" name="mapHeightSpinbox">
          <property name="minimum">
           <number>10</number>
          </property>
          <property name="maximum">
           <number>100000</number>
          </property>
          <property name="value">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item row="4" column="6">
         <widget class="QSpinBox" name="delaySpinbox">
          <property name="minimum">
           <number>10</number>
          </property>
          <property name="maximum">
           <number>7000000</number>
          </property>
          <property name="value">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item row="3" column="6">
         <widget class="QSpinBox" name="maxClientsSpinbox">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>100000</number>
          </property>
          <property name="value">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item row="3" column="5">
         <widget class="QLabel" name="maxClientsLabel">
          <property name="text">
           <string>Max clients per Team: </string>
          </property>
         </widget>
        </item>
        <item row="8" column="0" colspan="7">
         <widget class="Line" name="line2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="9" column="0" colspan="7">
         <widget class="QPushButton" name="launchServer">
          <property name="text">
           <string>Launch</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLineEdit" name="teamLineEdit"/>
        </item>
        <item row="5" column="1">
         <widget class="QPushButton" name="teamButtonAdd">
          <property name="text">
           <string>Add Team</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" rowspan="5" colspan="3">
         <widget class="QListWidget" name="teamList">
          <item>
           <property name="text">
            <string>Team1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Team2</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="5" column="5" colspan="2">
         <widget class="QRadioButton" name="verboseRadio">
          <property name="text">
           <string>Verbose mode</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="3" rowspan="6" colspan="2">
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="QPushButton" name="teamButtonClear">
          <property name="text">
           <string>Clear Teams</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="clientWidget">
     <attribute name="title">
      <string>Client</string>
     </attribute>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>9</y>
        <width>631</width>
        <height>351</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="2" column="0">
        <widget class="QLabel" name="teamLabel">
         <property name="text">
          <string>Team name: </string>
         </property>
        </widget>
       </item>
       <item row="8" column="0" colspan="2">
        <widget class="Line" name="line2_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="7" column="0" colspan="2">
        <widget class="QRadioButton" name="graphicRadio">
         <property name="text">
          <string>Launch Graphic Client</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="ipLineEdit">
         <property name="text">
          <string>127.0.0.1</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="ipLabel">
         <property name="text">
          <string>Machine Address: </string>
         </property>
        </widget>
       </item>
       <item row="9" column="0" colspan="2">
        <widget class="QPushButton" name="launchClient">
         <property name="text">
          <string>Launch</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="portLabel">
         <property name="text">
          <string>Machine Port: </string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="portSpinBox">
         <property name="maximum">
          <number>100000</number>
         </property>
         <property name="value">
          <number>8080</number>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="teamNameLineEdit">
         <property name="text">
          <string>Team1</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuZappy">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>669</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMainWindow">
    <property name="title">
     <string>Zappy</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionOpenMan"/>
    <addaction name="actionOpenQt"/>
    <addaction name="separator"/>
    <addaction name="actionOpenLog"/>
   </widget>
   <addaction name="menuHelp"/>
   <addaction name="menuMainWindow"/>
  </widget>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="actionNew_command">
   <property name="text">
    <string>New command</string>
   </property>
  </action>
  <action name="actionOpenMan">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>About Zappy</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionOpenLog">
   <property name="text">
    <string>Open Logs</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="actionAchievment">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Reach $1.000.000 to unlock</string>
   </property>
  </action>
  <action name="actionOpenQt">
   <property name="text">
    <string>About Qt</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
